lib/oli/lti/platform_external_tools.ex

```diff
@@ -197,7 +197,9 @@ defmodule Oli.Lti.PlatformExternalTools do
       end
 
     allowed_statuses =
-      [:enabled] ++ if options.include_disabled, do: [:disabled], else: []
+      [:enabled] ++
+        if(options.include_disabled, do: [:disabled], else: []) ++
+        if options.include_deleted, do: [:deleted], else: []
```

Comment:
NIT: Just a small thing. Instead of chaining multiple ++ operations, we could consider using `Enum.reduce` for a slightly more expressive approach like this

```
allowed_statuses =
  Enum.reduce([enabled: true, disabled: options.include_disabled, deleted: options.include_deleted], [], fn
    {status, true}, acc -> [status | acc]
    {_status, false}, acc -> acc
  end)
```